### ISSUE TITLE
SolarTimesSunrise and SolarTimesSunset are incorrect in the scheduler

### ISSUE THREAD

**Issue by couture999** [genielabs/HomeGenie#298]
We are back to Eastern Standard Time since 2 AM but the scheduler continue to show the Daylight Saving Time in the scheduler. So, all my scheduled events are offset by 1 hour.

**Comment by rbottomley** [comment]
I'm having the same issue now that we switched back to daylight saving time.

**Comment by Bounz** [comment]
@rbottomley  do you have correct time zone selected in your OS settings and last updates installed? HG is using `DateTime.ToLocalTime()` method and the exact output of this method depends on the current culture and the local time zone of the system on which it is run.

**Comment by rbottomley** [comment]
Yes I do have the time zone set properly and all of the latest updates. I just checked and noticed that tonight the lights will go on at the correct time. The bug is that sunset time is not calculated correctly on the day of the time switch, but after that it is OK.
